backend/backend/handlers/comments/commentService.py

```diff
@@ -0,0 +1,420 @@
+                    "assetVersionId:commentId": assetVersionIdAndCommentId,
+                }
+            )
+        except:
+            print(e)
+            response["statusCode"] = 400
+            response["message"] = e
+            return response
+
+        # Create a new comment with #deleted appended to the assetId
+        try:
+            table.put_item(Item=item)
+        except Exception as e:
+            print(e)
+            response["statusCode"] = 400
+            response["message"] = e
+            return response
+
+        response["statusCode"] = 200
+        response["message"] = "Comment deleted"
+    return response
+
+
+def set_pagination_info(queryParameters: dict):
+    """
+    Sets the pagination infor from the query parameters
+    :param queryParameters: dictionary containing pagination info
+    """
+    if "maxItems" not in queryParameters:
+        queryParameters["maxItems"] = 100
+        queryParameters["pageSize"] = 100
+    else:
+        queryParameters["pageSize"] = queryParameters["maxItems"]
+    if "startingToken" not in queryParameters:
+        queryParameters["startingToken"] = None
+
+
+def get_handler(response: dict, pathParameters: dict, queryParameters: dict) -> dict:
+    """
+    Function to handle the get request and route it to the right function
+    :param response: dictionary holding information about the response
+    :param pathParameters: dictionary holding information about the path (like versionId and/or assetId)
+    :param queryParameters: dictionary holding pagination information
+    :returns: Http response object (statusCode, headers, body) with comments stored in the body (if successful)
+    """
+    showDeleted = False
+
+    if "showDeleted" in queryParameters:
+        showDeleted = queryParameters["showDeleted"]
+
+    if "assetVersionId:commentId" not in pathParameters:
+        # if we have an assetVersionId and assetId, call get_comments_version
+        if "assetVersionId" in pathParameters and "assetId" in pathParameters:
+            print("Validating parameters")
+            (valid, message) = validate(
+                {"assetId": {"value": pathParameters["assetId"], "validator": "ID"}}
+            )
+            if not valid:
+                print(message)
+                response["body"] = json.dumps({"message": message})
+                response["statusCode"] = 400
+                return response
+
+            print(
+                "Listing comments for asset:",
+                pathParameters["assetId"],
+                "and version",
+                pathParameters["assetVersionId"],
+            )
+            response["body"] = json.dumps(
+                {
+                    "message": get_comments_version(
+                        pathParameters["assetId"],
+                        pathParameters["assetVersionId"],
+                        showDeleted,
+                    )
+                }
+            )
+            print(response)
```

Comment:
can skip this print statement, same for other branches below

Reply:
Fixed